packages/core/src/config/config.ts

```diff
@@ -0,0 +1,161 @@
+import { ConfigUrl } from '../index.js';
+import type {
+  Domain,
+  Domainlike,
+  EthereumConfig,
+  Network,
+  Resource,
+  SubstrateConfig,
+  SygmaConfig,
+} from '../types.js';
+import { Environment } from '../types.js';
+
+import { localConfig } from './localConfig.js';
+
+export class Config {
+  /**
+   * Cache storing all sygma
+   * environment configurations
+   */
+  configuration!: SygmaConfig;
+  initialized = false;
+
+  constructor() {}
```

Comment:
this is the same as default constructor. If we are not going to provide any arguments here we can ignore it AFAIK